samples/Domain/DomainContentTooLargeService.cs

```diff
@@ -0,0 +1,28 @@
+using System.Threading.Tasks;
+
+using DomainResults.Common;
+
+namespace DomainResults.Examples.Domain;
+
+public class DomainContentTooLargeService
+{
+	private const string Message = "Optimistic concurrency control: the record was just updated by someone else";
```

Comment:
If to be more realistic, then "_the request entity is larger than limits defined by server_" would be a better message